### PR TITLE
Add NOI Modifications

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: alcs-frontend/src/app/features/board/dialogs/noi-modification/create/create-noi-modification-dialog.component.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@use '../../../../../../styles/colors';
+
+.content {
+  padding: 0 16px;
+}
+
+.two-item-row {
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/board/dialogs/noi-modification/create/create-noi-modification-dialog.component.scss:7]: every file has it slightly different naming for the grid with 2 columns 😅

File: alcs-frontend/src/app/features/board/dialogs/noi-modification/create/create-noi-modification-dialog.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
+<div mat-dialog-title>
+  <h2 class="card-title">Create Modification</h2>
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/board/dialogs/noi-modification/create/create-noi-modification-dialog.html:2]: Create Modification -> Create NOI Modification?

File: alcs-frontend/src/app/features/notice-of-intent/decision/decision-dialog/decision-dialog.component.html [hunk 3]
```diff
@@ -12,7 +12,6 @@ <h2 class="card-title">{{ isEdit ? 'Edit' : 'New' }} Decision</h2>
           [matDatepicker]="datePicker"
           formControlName="date"
           name="date"
-          required
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/notice-of-intent/decision/decision-dialog/decision-dialog.component.html:15]: decision date is not required anymore?

File: services/apps/alcs/src/alcs/notice-of-intent-decision/notice-of-intent-modification/notice-of-intent-modification.controller.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      board,
+    );
+    return this.modificationService.mapToDtos([modification]);
+  }
+
+  @Delete('/:uuid')
+  @UserRoles(...ROLES_ALLOWED_APPLICATIONS)
+  async delete(@Param('uuid') uuid: string) {
+    await this.modificationService.delete(uuid);
+    return {};
```
Comment (mhuseinov) [inline services/apps/alcs/src/alcs/notice-of-intent-decision/notice-of-intent-modification/notice-of-intent-modification.controller.ts:62]: maybe something like    
 return {
      uuid,
    };

**Review by mhuseinov — APPROVED** [review]
🚀

File: alcs-frontend/src/app/features/notice-of-intent/decision/decision-dialog/decision-dialog.component.html (hunk 3 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/notice-of-intent/decision/decision-dialog/decision-dialog.component.html:15]: It is, but the required/not required should be set on the form control not the HTML when using Reactive Forms.

**Pull request merged**